src/main/java/com/icthh/xm/ms/entity/config/XmEntityTenantConfigService.java

```diff
@@ -59,6 +59,11 @@ public XmEntityTenantConfig getXmEntityTenantConfig() {
         return configs.get(tenantKey);
     }
 
+    public XmEntityTenantConfig getXmEntityTenantConfig(String tenantKey) {
+        configs.computeIfAbsent(tenantKey, (key) -> new XmEntityTenantConfig());
+        return configs.get(tenantKey);
```

Comment:
Don't need to do extra `get` operation because `computeIfAbsent` returns what you want